src/components/DayPicker.jsx

```diff
@@ -301,10 +305,12 @@ export default class DayPicker extends React.Component {
         didTransitionMonth = this.maybeTransitionPrevMonth(newFocusedDate);
         break;
       case 'Home':
+        if (e) e.preventDefault();
```

Comment:
Mostly for testing purposes, which like, could be done better. I can remove these in another PR.